homeassistant/components/tado/config_flow.py

```diff
@@ -136,6 +138,17 @@ async def async_step_import(self, import_config: dict[str, Any]) -> FlowResult:
             )
         except exceptions.HomeAssistantError:
             return self.async_abort(reason="import_failed")
+        except PyTado.exceptions.TadoWrongCredentialsException:
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                "failed_import_invalid_auth",
+                breaks_in_ha_version="2024.6.0",
+                is_fixable=False,
+                severity=IssueSeverity.ERROR,
+                translation_key="failed_import_invalid_auth",
+            )
```

Comment:
You're already raising this issue in `device_tracker.py` so you can remove it here

Reply:
That was silly, used for first debugging before passing through the abort reason... Good catch!

Reply:
Fix provided.